example/android-tutorial/img-segm-demo-app/semantic-segmentation-api/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.semantic_segmentation_api">
```

Comment:
I would suggest `io.singularitynet.sdk.tutorial` package as well

Reply:
applied in https://github.com/singnet/snet-sdk-java/pull/62/commits/79a2cf2865d96896b04868fe4f408407e9f969c9

Reply:
Here the package is still `com.example`

Reply:
fix is in https://github.com/singnet/snet-sdk-java/pull/62/commits/60108d6f1894fbfce96e03bcba853eda11400bfe